File: AppSelectorScreen/intentReciever/app/src/main/java/com/example/intentreciever/MainActivity.java
package com.example.intentreciever;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    private Button ebtn;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        ebtn=findViewById(R.id.Button);
        ebtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this,SecondPage.class);
                startActivity(intent);
            }
        });
    }
}

File: RoomDatabaseMoneyManagerapp/app/src/main/java/com/example/roomdatabasemoneymanagerapp/MainActivity.kt
package com.example.roomdatabasemoneymanagerapp

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.roomdatabasemoneymanagerapp.Fragments.PagerAdapter
import com.google.android.material.tabs.TabLayoutMediator
import kotlinx.android.synthetic.main.activity_main.*


class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        setAdapter()
        addActivity.setOnClickListener {
            val intent = Intent(this,AddActivity::class.java)
            startActivity(intent)
        }
    }
    private fun setAdapter() {
        val pagerAdapter = PagerAdapter(
            supportFragmentManager, lifecycle
        )
        viewPager.adapter = pagerAdapter
        TabLayoutMediator(
            tabLayout, viewPager
        ) { tab, position ->
            when (position) {
                0 -> tab.text ="Income"
                1 -> tab.text = "Expense"
                else -> tab.text = "Balance"
            }
        }.attach()
    }
}